app/web-app/src/app/modules/family/pages/family-data/family-data.component.html

```diff
@@ -78,8 +78,14 @@ <h3>Invite members</h3>
       </ng-container>
 
       <ng-container *ngIf="selectedTab==='requests'">
-
         <ng-container *ngIf="pendingRequests$ | async as pendingRequests">
+          <ng-container *ngIf="pendingRequests.length == 0">
+            <div class="row justify-content-center align-items-center vh-100">
+              <div class="col text-center">
```

Comment:
row and col classes don't exist. did you forget to add them or something else? Also, why do we use vh-100. flex-grow-1 should do just fine for a parent that can grow as well, no need for clamping it like that.